nucypher/characters/control/emitters.py

```diff
@@ -13,81 +13,64 @@ class StdoutEmitter:
 
         self.name = self.__class__.__name__.lower()
         self.sink = sink or self.default_sink_callable
-        self.capture_stdout = capture_stdout
-        self.quiet = quiet
+        self.verbosity = verbosity
         self.log = Logger(self.name)
 
-        super().__init__()
-
-    def __call__(self, *args, **kwargs):
-        try:
-            return self._emit(*args, **kwargs)
-        except Exception:
-            self.log.debug("Error while emitting nucypher controller output")
-            raise
-
-    def trap_output(self, output) -> int:
-        self.__stdout_trap.append(output)
-        return len(bytes(output))  # number of bytes written
-
-    def _emit(self,
-              response: dict = None,
-              message: str = None,
-              color: str = None,
-              bold: bool = False,
-              ) -> None:
-
-        """
-        Write pretty messages to stdout.  For Human consumption only.
-        """
-        if response and message:
-            raise ValueError(f'{self.__class__.__name__} received both a response and a message.')
-
-        if self.quiet:
-            # reduces the number of CLI conditionals by
-            # wrapping console output functions
-            return
-
-        if self.capture_stdout:
-            self.trap_output(response or message)
-
-        elif response:
-            # WARNING: Do not log in this block
+    def clear(self):
+        if self.verbosity >= 1:
+            click.clear()
+
+    def message(self,
+                message: str,
+                color: str = None,
+                bold: bool = False,
+                verbosity: int = 1):
+        self.echo(message, color=color or self.default_color, bold=bold, verbosity=verbosity)
+        self.log.debug(message)
+
+    def echo(self,
+             message: str,
+             color: str = None,
+             bold: bool = False,
+             nl: bool = True,
+             verbosity: int = 0):
+        if verbosity <= self.verbosity:
+            click.secho(message=message, fg=color or self.default_color, bold=bold, nl=nl)
+
+    def banner(self, banner):
+        if self.verbosity >= 1:
+            click.echo(banner)
+
+    def ipc(self, response: dict, request_id: int, duration):
+        # WARNING: Do not log in this block
+        if self.verbosity >= 1:
             for k, v in response.items():
-                click.secho(message=f'{k} ...... {v}',
-                            fg=color or self.default_color,
-                            bold=bold)
-
-        elif message:
-            # Most likely a message emitted without a character control instance
-            click.secho(message=message, fg=color or self.default_color, bold=bold)
-            self.log.debug(message)
+                click.secho(message=f'{k} ...... {v}', fg=self.default_color)
 
-        else:
-            raise ValueError('Either "response" dict or "message" str is required, but got neither.')
+    def error(self, e):
```

Comment:
I would prefer to avoid that, see above (and TODO in the PR description). I would rather eliminate all explicit style specification from the code, since it is hard to synchronize it between people, and you can never be sure whether to use red, or green, or some other color in some specific case - semantic calls like `error()` or `warning()` are more clear.